packages/common/src/logger/logger.ts

```diff
@@ -15,11 +15,12 @@ export class Logger {
   private pino: Pino.Logger;
   private childLoggers: {[category: string]: Pino.Logger} = {};
 
-  constructor({level: logLevel, nestedKey, outputFormat}: LoggerOption) {
+  constructor({level, nestedKey, outputFormat}: LoggerOption) {
```

Comment:
It is intended using alias logLevel, to avoid variable naming conflict. e.g at line 62